pkg/updater/updater.go

```diff
@@ -533,7 +533,9 @@ func InflateDropAppend(ctx context.Context, alog logrus.FieldLogger, client gcs.
 	case <-ctx.Done():
 		return false, fmt.Errorf("first column: %w", ctx.Err())
 	case col := <-newCols:
-		cols = append(cols, col)
+		if len(col.Cells) != 0 {
```

Comment:
A concern I have with this approach is that this will prevent us from updating the hint....

What would you think about keeping around the most recently received empty column to mitigate this?

Reply:
OK I guess you specifically call that out in the PR description but I've been fighting a lot of hint-related issues so I'm a bit concerned.

Reply:
PS: I'd do

```golang
if len(col.Cells) == 0 {
  continue
}
cols = append(cols, col)
```

Reply:
Discussed offline, and implemented the "group empty columns into one empty column" idea.